orchagent/lagids.lua

```diff
@@ -0,0 +1,88 @@
+-- KEYS - None
```

Comment:
Shouldn't we add this new lagids.lua  into Makefile ?

Reply:
Yes. It is added in the make file when it is used. In this PR, we just implement the lag id allocator. We are not using it yet. In PR #1605 we update the make file and use the lag id allocator.